tests/cli/test_cmds_logs.py

```diff
@@ -94,14 +94,24 @@ def test_paasta_log_line_passes_filter_true():
     clusters = ["fake_cluster1", "fake_cluster2"]
     instance = "fake_instance"
     instances = [instance]
+    pod = "fake_pod"
+    pods = [pod]
     components = ["build", "deploy"]
     line = "fake_line"
     formatted_line = format_log_line(
-        levels[0], clusters[0], service, instance, components[0], line
+        levels[0], clusters[0], service, instance, components[0], line,
     )
+
+    # LogWriters don't write logs for pods yet. Since LogWriters use format_log_line, we're
```

Comment:
I don't think you need to do any of this for the paasta_log_line tests. We expect that paasta_log_line will be filtering lines that do not have pod_name in them (remember, paasta_log_line is all the components that are not app_output). 

You should just be able to set pods = None and the test should pass.

Reply:
Done.